wordpress-mobile/WordPress-iOS
# Break gravatar caching on signup

Fixes a problem where a gravatar uploaded during signup wouldn't appear until the app was relaunched.

Now it works:

![hack week gravatar fix](https://user-images.githubusercontent.com/517257/44968141-e57c8480-aefa-11e8-99fb-ff3edd47bbcc.gif)


To test:
- signup for a new `wpcom` account
- upload a gravatar
- flip to the me tab and see if your new gravatar is there
